notebooks/0-the-cloud/0-the-cloud.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>query_criteria is the most general, since you can search on the maximum number of criteria. (i.e., can get all JWST obs, all spectra in the archive, etc.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I updated this text already, not sure why the old version is still here ?</p>